content/handbook/product/categories/_index.md

```
@@ -0,0 +1,435 @@
+
+私たちは、一部のアナリストが SAST と Code Quality もこれらのカテゴリに含めるためにこの用語を使用しているにもかかわらず、[SCA を SAST および Code Quality を含むものとして定義しないことを意図的に決めています](https://gitlab.com/gitlab-com/www-gitlab-com/merge_requests/26897#note_198503054)。
+
+### Capabilities
+
+Capability は、ステージ、カテゴリ、または機能を指すことができますが、ソリューションは指しません。
+
+### レイヤー
+
+階層にレイヤーを追加することで詳細度は増しますが、以下の点でユーザビリティを損ねます:
+
+1. [インターフェース](#interfaces) を最新に保つのが難しくなる。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)